ocaml/idl/datamodel.ml

```diff
@@ -2389,6 +2389,19 @@ let vm_call_plugin = call
 	~allowed_roles:_R_VM_OP
 	()
 
+let vm_import_from = call
+	~name:"import_from"
+	~in_product_since:rel_dundee
+	~doc:"Import an XVA from a URI"
+	~params:[String, "url", "The URL of the XVA file";
```

Comment:
I find `import_from` a bit vague... Can we make it something like `import_from_url`? Or simple `import`, since we don't actually have that as an API call.

Also, the doc string says URI, but the param has URL.